### ISSUE TITLE
question about oputput

### ISSUE THREAD

**Issue by Annabelle115** [biboamy/instrument-prediction#6]
Is another dimension of the output npy file a frame?

**Comment by Annabelle115** [comment]
can you tell me how can I get the ./json file,how can I  transfer the predict result to time

**Comment by biboamy** [comment]
For output npy, first dimension represents each instrument, second dimension represents time frame. For the time, I currently generate 9 labels each second, so each data frame represents (1/9) second.

**Comment by Annabelle115** [comment]
the second demension of 1759.nty is 1800,equal to 200s.But the duration of 1759.wav is 194.6 according to librosa.get_duration.....

**Comment by biboamy** [comment]
Yes, that is because for the current testing method, I chuck the input data into 10 seconds. For the last chuck which is not enough for 10 seconds, I zero-pad to 10 seconds. So when you get the output, feel free to discard the zero-padding part. 

However, I know it's not necessary to chuck the data. So please also feel free to modify the testing code. Hope this is clear!

**Comment by Annabelle115** [comment]
thanks~~~
